studio/components/interfaces/BillingV2/AddNewPaymentMethodModal/AddNewPaymentMethodModal.tsx

```diff
@@ -52,7 +66,7 @@ const AddNewPaymentMethodModal = ({
           return
         }
 
-        await setupIntent(token ?? undefined)
+        await initSetupIntent(token ?? undefined)
```

Comment:
Can you define `initSetupIntent` _before_ this `useEffect` and then call it via `useLatest`? Or put it as an inline function inside the `useEffect`

Reply:
actually yeah i'll shift it inside the useEffect since its only used in that block

Reply:
RE pausing polling - hmm super odd i remember there used to be some polling logic but i actually can't find it now too, not part of the changes in the PR as well. i'll reinstate it

Reply:
@alaister just pushed - added long polling logic with RQ in ProjectPausingState